src/multidim/setdiff.jl

```diff
@@ -105,3 +105,25 @@ end
 #
 #     return results_list
 # end
+
+import IntervalArithmetic.hull
+doc"""
+    hull(a, b)
+
+Returns the `box hull` of the IntervalBoxes `a` and `b`, considered as
+(extended) $N$ dimensional sets of real numbers, i.e. the smallest region that contains
+all of `a` and `b`.
+"""
+function hull(A::IntervalBox{N,T}, B::IntervalBox{N,T}) where {N,T}
+    result_list = Vector{Interval{T}}()
+
+    global_lo = convert(T,0)
+    global_hi = convert(T,0)
+
+    for dimension in 1:N
+        global_lo = min(Base.getindex(A,dimension).lo,Base.getindex(B,dimension).lo)
```

Comment:
Note that `Base.getindex(A, dimension)` can (and should) be written as just `A[dimension]`.